src/main/java/hudson/remoting/InternalCallable.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2021, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.remoting;
+
+/**
+ * For remoting internal use only: {@link hudson.remoting.Callable}s implement
+ * this marker interface to be allowed to bypass the required role check of
+ * {@link hudson.remoting.RequiredRoleCheckerWrapper}.
+ *
+ * @since TODO
```

Comment:
```suggestion
```

Not an API, so no need for `@since`.

And add `@Restricted(NoExternalUse.class)`.

Reply:
I suggest keeping `@since`, may be useful even for internal stuff.

Reply:
Generally prefer not to include it, since `git blame` is your friend for tracking behavioral changes if you are already inside the source tree. Not important either way.